### ISSUE TITLE
use Colons instead of Backticks 

### ISSUE THREAD

**Issue by VolkerLemke** [rafaelprd/azure-devops-markdown-mermaid#2]
In Azure Devops usually colons are used to surround mermaid diagrams in markdown files. Espacially in the Wiki colons are mandatory because backticks do not work there. Can you add the recognition of colons to the plugin so its behavior is identical to the Azure Devops default and the same syntax (and files) can be used in all locations?

**Comment by rafaelprd** [comment]
Yes, it really makes sense recognition of colons. I'll try to add it.

**Comment by StingyJack** [comment]
Please dont require `:::`. Many of us arent making AzDo flavored markdown and will still need the triple backtick (and markdown tables - all of these mermaid rendering extensions seem to cause markdown tables to no longer render properly)
